client/signup/style.scss

```
@@ -425,3 +425,73 @@ body.is-section-signup.is-new-launch-flow .layout:not( .dops ):not( .is-wccom-oa
 		color: var( --color-text );
 	}
 }
+
+/**
+ * Common styles for reskinSignupFlow a/b test
+ */
+body.is-section-signup.is-white-signup .signup.is-onboarding {
+	$gray-100: #101517;
+	$gray-60: #50575e;
+	$gray-50: #646970;
+	$breakpoint-mobile: 660px;
+
```

```suggestion
--color-accent: #117ac9;
```
Can we also add this here? It will automatically solve a lot of highlighting/active color issues in the other PRs.

---

@aneeshd16 `--color-accent` is introduced by d1bae34b9e9d626ed9540cd47b027f8da5ae7b32